grantnav/frontend/templates/funder.html

```diff
@@ -1,194 +1,228 @@
-{% extends 'base.html' %}
-
-{% load frontend %}
-{% load static %}
-
+{% extends "base.html" %} 
+{% load frontend %} 
+{% load static %} 
 {% block main_content %}
-
-<div class="row">
-  <div class="col-md-4">
-    <h1 class="funder-header text-center">
-      {{ funder.name }}
-    </h1>
-   
-  </div>
-  <div class="col-md-8">
-    <div class="row">
-      <div class="col-xs-6">
-        <dl class="dl-horizontal top-stats">
-          <dt>Total grants</dt> <dd>{{results.hits.total|get_amount}}</dd>
-          <dt>Total recipients</dt> <dd>{{results.aggregations.recipient_orgs.value|get_amount}}</dd>
-          <dt>Earliest award date</dt> <dd>{{results.aggregations.min_date.value_as_string|get_date}}</dd>
-          <dt>Latest award date</dt> <dd>{{results.aggregations.max_date.value_as_string|get_date}}</dd>
-        </dl>
+<div class="layout layout--single-column">
+  <main class="layout__content">
+    <div class="grid grid--two-columns spacer-4">
+      <div class="grid__1">
+        <h1 class="funder-header text-center">{{ funder.name }}</h1>
       </div>
-      <div class="col-xs-6">
-        {% include 'currency_stats.html' %}
+      <div class="grid grid--two-columns grid__1">
+        <div class="grid__1">
+          <dl class="top-stats">
```

Comment:
It would be nice if these tables looked a little nicer, but perhaps not a blocker and one to pick up on the other side (it is quite messy though). We _could_ split the `<dt>` and `<dd>` elements by line rather than wrapping them on the same line.

Reply:
It'll likely take up more space but I think look a little more polished... possibly. Until we get the new summary stats box in place

Reply:
Need a concrete action on resolving this if it is a blocker, otherwise we can do this as an incremental improvement after launch.